doc/contribution-guidelines.md

```diff
@@ -0,0 +1,97 @@
+# Contribution guidelines
+
+- [Pull Requests](#pullrequest)
+- [Commit Message Guidelines](#commit)
+
+## <a name="pullrequest"></a> Pull Requests Guidelines
+Pull requests are the only means by which you can contribute to this project, please follow the following steps when submitting pull requests :
+
+1. Create your pull request. (PR name must match [commit name](#commit))
```

Comment:
The first step, creating the PR from the users fork with the PR name respecting the commit format name.